csharp/test/Drivers/Snowflake/ClientTests.cs

```diff
@@ -221,6 +221,86 @@ public void VerifyTypesAndValues()
             }
         }
 
+        [Test, Order(6)]
+        public void CanClientExecuteStageCommand()
```

Comment:
What do the stage and copy commands add that you can't do with the existing `Query` value? Instead of being configurable with a different test configuration (set via the environment variable), it now adds separate queries and has a fixed result (1) instead of being configurable.

Reply:
In our current architecture, changing environment variables to switch config file is necessary for different test cases. Maybe we could do some refactor to decouple the query and authentication config.  This could eliminate the need for multiple config files and switching environment variables.

This is my proposal.
- The config file should only support different auth type with auth credentials.
- In each test, we can have some data prepare before test case run. And store query and expected result in separate place.

Reply:
This is a good idea. I know the auth portion can get tedious with multiple files.